src/rpc/mempool.cpp

```diff
@@ -85,13 +85,11 @@ static RPCHelpMan sendrawtransaction()
                                                      DEFAULT_MAX_RAW_TX_FEE_RATE :
                                                      CFeeRate(AmountFromValue(request.params[1]));
 
-            int64_t virtual_size = GetVirtualTransactionSize(*tx);
-            CAmount max_raw_tx_fee = max_raw_tx_fee_rate.GetFee(virtual_size);
 
             std::string err_string;
             AssertLockNotHeld(cs_main);
             NodeContext& node = EnsureAnyNodeContext(request.context);
-            const TransactionError err = BroadcastTransaction(node, tx, err_string, max_raw_tx_fee, /*relay=*/true, /*wait_callback=*/true);
+            const TransactionError err = BroadcastTransaction(node, tx, err_string, /*max_tx_fee*/0, max_raw_tx_fee_rate, /*relay=*/true, /*wait_callback=*/true);
```

Comment:
in "[node]: update BroadcastTransaction to check fee rate limit" (https://github.com/bitcoin/bitcoin/commit/eedbeccbdb274841fc2cf2c53931e58fbea50dfc):

nit:
```suggestion
            const TransactionError err = BroadcastTransaction(node, tx, err_string, /*max_tx_fee=*/0, max_raw_tx_fee_rate, /*relay=*/true, /*wait_callback=*/true);
```

Reply:
Fixed